igvm/vm.py

```diff
@@ -241,7 +242,8 @@ def aws_start(self):
         Start a VM in AWS.
         """
 
-        ec2 = boto3.client('ec2')
+        session = boto3.Session(profile_name=AWS_PROFILE_NAME)
```

Comment:
Does it make sense to create the session once and store it as instance variable and use a getter to create it the first time and otherwise return it to avoid creating a new session every time in a run ?

Reply:
You're absolutely right. I reworked it using a property for the session initialisation.